docs/dashboards/subscriptions.md

```diff
@@ -27,6 +27,13 @@ Let's say we want to email a dashboard. We'll click on the **Email it** option i
 
 ![Dashboard subscription email options](./images/email-options.png)
 
+## How permissions work with dashboard subscriptions
+
+- **Recipients of dashboard subscriptions will be able to see whatever the creator of the dashboard subscription can see.** That is, people will get to see charts in their email or Slack _as if_ they had the subscription creator's permissions to view those charts, _regardless of whether their groups have permission to view those charts_.
+- **Admins can modify subscriptions without affecting its permissions.** Admins can see all dashboard subscriptions. But here's a subtle point about how permissions work out: if someone creates a dashboard subscription, an admin can modify that subscription without interfering with the permissions associated with that subscription. Meaning: if an admin updates a dashboard subscription, the dashboard subscription doesn't suddenly get admin permissions; the permissions remain the same. So if a subscription creator can't view a chart on a dashboard, an admin can modify that subscription without making that chart visible for the subscription's creator, or its recipients. 
```

Comment:
I'm not sure why, but I think this is a bit confusing.

I would say something like

**Admins can see and edit all subscriptions**
Admins can modify any subscription (alter recipients, filters, or delete the subscription). But the subscription will continue to send data based on the subscription creator's permissions.